cockpit/leapp.html

```diff
@@ -241,6 +296,61 @@ <h4 class="text-center">Application names already in use:</h4>
             $(".analyze").fadeIn();
         });
 
+        $("#map-new-port-show").click(function(){
+            $("#map-new-port-form").find("input[type=text]").val("")
```

Comment:
please use semicolons, like everywhere in the code, probably we should use strict mode. Maybe it's a good idea to assign #map-new-port-form to some var?